### PR TITLE
Add queries and mutations for VRT Conversion

### PR DESCRIPTION
**Pull request by coreyar**
Adds queries and mutations needed to connect the new VRT Conversion screen

### CODE REVIEW

File: src/clients/api/mutations/useApproveVrt.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        vrtTokenContract,
+        ...params,
+      }),
+    {
+      ...options,
+    },
+  );
+};
+
+export default useApproveVrt;
```
Comment (therealemjy) [inline src/clients/api/mutations/useApproveVrt.ts]: Can't we use `useApproveToken` directly?

File: src/clients/api/queries/useGetXvsBalanceOf.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  const xvsTokenContract = useTokenContract('xvs');
+
+  return useQuery(
+    FunctionKey.GET_XVS_BALANCE_OF,
+    () => getXvsBalanceOf({ xvsTokenContract, accountAddress }),
+    options,
+  );
+};
+
+export default useGetXvsBalanceOf;
```
Comment (therealemjy) [inline src/clients/api/queries/useGetXvsBalanceOf.ts]: I'd suggest creating a `useGetTokenBalance` instead, similarly to `useGetVTokenBalance`, that can then be used to get the balances for XVS and VRT (and any other token we support that's not a Venus token).

File: src/clients/api/queries/useGetVrtAllowance.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  const vrtTokenContract = useTokenContract('vrt');
+
+  return useQuery(
+    FunctionKey.GET_VRT_TOKEN_ALLOWANCE,
+    () => getVrtAllowance({ vrtTokenContract, accountAddress, spenderAddress }),
+    options,
+  );
+};
+
+export default useGetVrtAllowanceIndex;
```
Comment (therealemjy) [inline src/clients/api/queries/useGetVrtAllowance.ts]: Same thing here, I'd create a `useGetTokenAllowanceIndex` that's used for all tokens

File: src/clients/api/mutations/useApproveVrt.ts (hunk 1 above)
Comment (coreyar) [reply src/clients/api/mutations/useApproveVrt.ts]: We could. The difference is the contract it uses

**Review by therealemjy — APPROVED** [review]
Thanks for the updates.
I've approved but feel free to message me if you want to discuss why I requested the changes more, since yesterday I asked you if we could talk about this later given we had the release.

**Comment by coreyar** [comment]
> Thanks for the updates. I've approved but feel free to message me if you want to discuss why I requested the changes more, since yesterday I asked you if we could talk about this later given we had the release.

All good. I saw what you were pointing out

**Pull request merged**